lib/common.js

```diff
@@ -154,11 +154,13 @@ function stringifyRequest(options, body) {
   let { port } = options
   // TODO-coverage: Add a test to cover the missing condition, or remove if
   // not reachable.
-  if (!port) port = options.proto == 'https' ? '443' : '80'
+  // The step seems useless considering we don't show the port for standard 80/443 anyway.
+  // The net affect this has is to default the port to 80 when the proto *is not* http. (which probably isn't right)
```

Comment:
```suggestion
  // It's not clear whether the port could be set to something other than http or https. If
  // it is, this would make an unspecified port default to 80, which is probably incorrect.
```